soc/realtek/ec/rts5912/reg/reg_tacho.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2025 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_TACHO_H
+#define ZEPHYR_SOC_REALTEK_RTS5912_REG_TACHO_H
+
+typedef struct {
+	union {
+		volatile uint32_t CTRL;
+
+		struct {
+			volatile uint32_t EN: 1;
+			volatile uint32_t FILTEREN: 1;
+			volatile uint32_t SELEDGE: 2;
+			volatile uint32_t READMODE: 1;
+			uint32_t: 11;
+			const volatile uint32_t CNT: 16;
+		} CTRL_b;
+	};
+
+	union {
+		volatile uint32_t STS;
+
+		struct {
+			volatile uint32_t LIMIT: 1;
+			const volatile uint32_t PIN: 1;
+			volatile uint32_t CHG: 1;
+			volatile uint32_t CNTRDY: 1;
+			uint32_t: 28;
+		} STS_b;
+	};
+
+	union {
+		volatile uint32_t LIMITH;
+
+		struct {
+			volatile uint32_t VAL: 16;
+			uint32_t: 16;
+		} LIMITH_b;
+	};
+
+	union {
+		volatile uint32_t LIMITL;
+
+		struct {
+			volatile uint32_t VAL: 16;
+			uint32_t: 16;
+		} LIMITL_b;
+	};
+
+	union {
+		volatile uint32_t INTEN;
+
+		struct {
+			volatile uint32_t LIMITEN: 1;
+			volatile uint32_t CNTRDYEN: 1;
+			volatile uint32_t CHGEN: 1;
+			uint32_t: 29;
+		} INTEN_b;
+	};
+} TACHO_Type;
+
+/* CTRL */
+#define TACHO_CTRL_EN_Pos        (0UL)
+#define TACHO_CTRL_EN_Msk        BIT(TACHO_CTRL_EN_Pos)
```

Comment:
Simplify your single-bit field definitions.
```suggestion
#define TACHO_CTRL_EN        BIT(0)
```

Reply:
done